src/libraries/Common/src/System/Security/Cryptography/SecKeyPair.cs

```diff
@@ -11,11 +11,15 @@ internal sealed class SecKeyPair : IDisposable
     {
         internal SafeSecKeyRefHandle PublicKey { get; private set; }
         internal SafeSecKeyRefHandle? PrivateKey { get; private set; }
+        internal SafeSecKeyRefHandle? PublicDataKey { get; private set; }
+        internal SafeSecKeyRefHandle? PrivateDataKey { get; private set; }
 
-        private SecKeyPair(SafeSecKeyRefHandle publicKey, SafeSecKeyRefHandle? privateKey)
+        private SecKeyPair(SafeSecKeyRefHandle publicKey, SafeSecKeyRefHandle? privateKey, SafeSecKeyRefHandle? publicDataKey, SafeSecKeyRefHandle? privateDataKey)
```

Comment:
Actually, now I'm confused.  If the existing keys were SecKeyRefs, and the new keys are SecKeyRefs, what's the difference, and why don't they interoperate?

I'm not finding the data in the docs

Reply:
It's... hard to articulate and compounded by I think there is at least one bug in MacOS that I am trying to get chased down. I'll experiment a bit and come back with a more thorough explanation of things.

Now that I have Big Sur working and runtime building on it (surprise, nothing in S.S.C broke so far) I wonder if I will still hit quirks.